roles/kiwix/tasks/install.yml

```
@@ -44,32 +44,22 @@
     path: "{{ item }}"
     state: directory
   with_items:
+    - "{{ kiwix_path }}/bin"    # /opt/iiab/kiwix
     - "{{ iiab_zim_path }}"    # /library/zims
     - "{{ iiab_zim_path }}/content"
     - "{{ iiab_zim_path }}/index"
 
-- name: Check for {{ kiwix_library_xml }}
+- name: Check for {{ iiab_zim_path }}/content/*
```

Line 52 and 54 should probably test for *.zim suffix to be a bit safer?

```suggestion
- name: Check for {{ iiab_zim_path }}/content/*.zim
```

---

done